R/g.dotorcomma.R

```diff
@@ -26,7 +26,7 @@ g.dotorcomma = function(inputfile, dformat, mon, desiredtz = "", loadGENEActiv =
     # lot of zeros, which makes it impossible to detect decimal separator
     # "." will then be the default, which is not correct for "," systems.
     while (skiprows < 1000000) { #foundnonzero == FALSE & 
-      deci = as.matrix(read.csv(inputfile, skip = skiprows, nrow = 10))
+      deci = as.matrix(data.table::fread(inputfile, skip = skiprows, nrows = 10, data.table = FALSE))
       skiprows = skiprows + 10000
```

Comment:
Please undo this change to g.dotorcomma, as we discussed [here](https://github.com/wadpac/GGIR/pull/791#issuecomment-1566764332). It has taken a lot of time to develop and test the code for reading csv files. Changing it without addressing a clear bug and without a clear functional improvement seems like an unnecessary risk.